src/js/components/receiving/PartialReceivingPage.jsx

```diff
@@ -253,10 +253,21 @@ const TABLE_FIELDS = {
           showValueTooltip: true,
         },
       },
+      uom: {
+        type: (params) => <LabelField {...params} />,
+        label: 'react.partialReceiving.shippedInPo.label',
+        defaultMessage: 'Shipped (in PO)',
+        multilineHeader: true,
+        flexWidth: '1',
+        getDynamicAttr: (props) => ({
+          hide: !props.values?.isShipmentOrderBased,
+        }),
+      },
```

Comment:
shouldn't this field be disabled?

Reply:
I think LabelField is "disabled" by default